integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMultiDomainModelsWithLoadBalancer.java

```diff
@@ -358,19 +303,22 @@ void testScaleClustersByPatchingDomainResource(Domain domain) {
 
     stopPortForwardProcess(domainNamespace);
 
+    // shutdown domain and verify the domain is shutdown
+    shutdownDomainAndVerify(domainNamespace, domainUid, replicaCount);
   }
 
   /**
    * Scale cluster using REST API for three different type of domains.
    * i.e. domain-on-pv, domain-in-image and model-in-image
    *
-   * @param domain oracle.weblogic.domain.Domain object
+   * @param domainType domain type, possible value: modelInImage, domainInImage, domainOnPV
    */
   @ParameterizedTest
   @DisplayName("scale cluster using REST API for three different type of domains")
-  @MethodSource("domainProvider")
-  void testScaleClustersWithRestApi(Domain domain) {
-    assertDomainNotNull(domain);
+  @ValueSource(strings = {"modelInImage", "domainInImage", "domainOnPV"})
```

Comment:
Can we make these methods ( testScaleClustersWithRestApi, testScaleClustersWithWLDF,testAdminConsoleLoginUsingAdminNodePort,testAdminConsoleLoginUsingIngressController)  as local test utility  method and call them from a single test method.  This will reduce the number of time we crate same domain in multiple test method.

Reply:
changed testAdminConsoleLoginUsingAdminNodePort() and testAdminConsoleLoginUsingIngressController() to utility methods and added them to other parameterized tests.